### PR TITLE
[dev] AB#60935 Information on the table is being overlapped

### PR DESCRIPTION
**Pull request by IamNotHuman**
added callback on ActionBar to detect when is mobile input selected or not

### CODE REVIEW

**Review by groberts314 — APPROVED** [review]
LGTM

**Review by jeoffm — APPROVED** [review]

File: src/surfaces/actionBar/actionBarSearch.jsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ const defaultProps = {
     isMobileSearch: false,
     isMobileSearchVisible: false,
     onClearClick: undefined,
+    onFocused: undefined,
```
Comment (cbrewer1689) [inline src/surfaces/actionBar/actionBarSearch.jsx:28]: So do we deliberately want past tense here?

Comment (groberts314) [reply src/surfaces/actionBar/actionBarSearch.jsx:28]: No, we usually use present tense for this kinds of things, right?  I think request a change; should be easy enough.

Comment (IamNotHuman) [reply src/surfaces/actionBar/actionBarSearch.jsx]: renamed

Comment (groberts314) [reply src/surfaces/actionBar/actionBarSearch.jsx]: Muchas gracias! =)

**Review by cbrewer1689 — APPROVED** [review]
Looking good!

**Pull request merged**